engine/src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -197,25 +197,39 @@ trait CromwellApiService {
       case b: BodyPart => b.toStrict(duration).map(strict => b.name -> strict.entity.data)
     }.runFold(Map.empty[String, ByteString])((map, tuple) => map + tuple)
 
+    def toResponse(workflowId: WorkflowId): WorkflowSubmitResponse = {
+      WorkflowSubmitResponse(workflowId.toString, WorkflowSubmitted.toString)
+    }
+
+    def askSubmit(command: WorkflowStoreActor.WorkflowStoreActorSubmitCommand): Route = {
+      // NOTE: Asks were already here. Guessing this was a KISS port of per-request via:
```

Comment:
Why comment it? That's literally what the [docs say](http://doc.akka.io/docs/akka-http/current/scala/http/routing-dsl/index.html), what their devs say, and blogs supporting the per-request pattern are just hacks on an old and overly cargo culted pattern. The per-request thing was not due to the fact that the underlying computation might be happening via actors, it was solving a different problem that almost no one (including us) really had. Not sure yow this is possible but apparently the Spray developers themselves were unaware of per-request, so it was only "spray-esque" in that a lot of people were copypasta'ing the pattern on their own.

Also, everything else in this file that's talking to an actor is using an ask. If you want to point out something as being a KISS artifact of our per-request past, they exist in this file (`metadataBuilderActor` I'm looking at you - was too hairy to decompose). So for instance I did look at unactoring the metadataBuilderActor because in that case a new one is being constructed for every call, but with all the FSM states in there I just didn't want to bother.